.lintstagedrc.json

```diff
@@ -1,5 +1,7 @@
 {
-  "**/*.js": [
-    "eslint --fix"
```

Comment:
Just for anyone who's curious, I made this change because otherwise the linter would get upset at the JS bundled into the docs site.

Reply:
what i might recommend instead is updating `.eslintignore` to add `docs/scripts/*`. then we don't have to bend over backwards to ignore things.

what you have works, and there are a few other styles possible, but reading through their documentation it seems their opinion is to push ignoring of files into the specific tooling you are using.

https://www.npmjs.com/package/lint-staged#ignoring-files

> and ignoring any files should be configured in the task itself.